package.json

```diff
@@ -43,7 +43,6 @@
     "applicationinsights": "^1.8.7",
     "autobind-decorator": "^2.4.0",
     "awilix": "^4.2.6",
-    "axios": "^0.20.0",
```

Comment:
I think I might need this for the changes in https://github.com/hmcts/fact-frontend/pull/70

Reply:
there is already another axios in the file hence removed this as duplicate

Reply:
ill remove the other one for alphabetic ordering